pkg/core/resources/apis/mesh/gateway_route_validator_test.go

```diff
@@ -649,6 +649,31 @@ conf:
         - weight: 5
           destination:
             kuma.io/service: target-2
+`),
+		ErrorCases("prefix with trailing slash", []validators.Violation{{
```

Comment:
we have a case with a prefix and a suffix, don't we need two test for these?

Reply:
This case tests that, that's why we get two errors. Is that what you mean? I'll make the name clearer.

Reply:
Did not notice that we have 2 errors, if you can adjust the name that would be great.